### ISSUE TITLE
npm test fails

### ISSUE THREAD

**Issue by aral** [fanout/reconnecting-eventsource#19]
```sh
> reconnecting-eventsource@1.0.3 test /home/aral/small-tech/site.js/forks/reconnecting-eventsource
> node test

internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module '.'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/home/aral/small-tech/site.js/forks/reconnecting-eventsource/test.js:2:33)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
npm ERR! Test failed.  See above for more details.
```

**Comment by aral** [comment]
This is because it needs to be built first. This should be documented.

To run tests:

```sh
npm run-script build
npm test
```

**Comment by jkarneges** [comment]
Fixed in c165f990f81ed76b24291d5033343f2cc875d99e from #20.
